src/CodeReview.Orchestrator/Model/Volume.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using YamlDotNet.Serialization;
+
+namespace GodelTech.CodeReview.Orchestrator.Model
+{
+    public class Volume : ICloneable
```

Comment:
@pNovikau formatting of this file looks strange. I would suggest to put attribute name on separate line. E.g.:

```c#
[YamlIgnore] 
public string Name { get; set; }
```

Please apply this pattern to all attributes